community/hacktoberfest/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: single
+title:  "Hacktoberfest"
+sidebar:
+  nav: community
+---
+
+[Hacktoberfest](https://hacktoberfest.digitalocean.com/) is a month-long global celebration of open source software that happens every year in October. During this event, anyone can support open source by contributing changes, which earn swag and recognition from the [Continuous Delivery Foundation](https://cd.foundation), our parent organization.
```

Comment:
There's a little confusion in this paragraph about Hacktoberfest being bigger than just Spinnaker and the CDF, but swag coming from CDF for Spinnaker contribs. The confusion is in the phrase "During this event," which refers to Hacktoberfest. 

Maybe: "During this event, you can contribute to Spinnaker and earn swag and recognition from the [CDF]..."

Reply:
Good suggestion. I ended up getting rid of the word "event" to avoid any confusion. It just says "During the month of October" now since...it isn't really an event.